_posts/2024-06-07-opensearch-performance-2.14.md

```diff
@@ -0,0 +1,161 @@
+- Reducing the amount of space needed to store vectors by compressing vector components from 32 bits to 16, 8, and even 1 bit. Our experiments show modest reductions in search quality (recall) with 16 and 8 bit components; we are still analyzing the 1-bit case (binary quantization).
+- Reducing cost by employing disk-based ANN algorithms that use external storage (SSDs, S3, and so on).
+
+Vector techniques have also been less flexible than lexical techniques in many ways, so we are:
+
+- Improving search relevancy by supporting reranking on compressed vectors.
+- Supporting ANN vector search for the multi-tenant case, where each tenant has its own subcollection. This is especially valuable for customers each of whose user organizations has a large vector collection.
+
+### Performance benchmarking
+
+We are planning to update [the Big5 workload](https://github.com/opensearch-project/opensearch-benchmark-workloads/tree/main/big5) mappings and settings. In particular, we'll add support for dynamic templates, configurable merge policies, match-only text, and setting the default query field as `message`.
+
+## Conclusion
+
+While we're continuing to expand OpenSearch functionality, we are also investing in improving the performance, efficiency, and robustness of the system for every workload. In particular, vector techniques are being widely adopted both for pure semantic search and for hybrid semantic and lexical search.
+
+The OpenSearch team at AWS works in collaboration with the larger OpenSearch community. Without its contributions to testing, feedback, and development, OpenSearch would not be where it is today.
+
+Stay tuned to our blog and GitHub for continuous updates and insights into our progress and future plans.
+
+## Appendix: Benchmark tests and results
+
+This section presents details on the performance benchmarks we used and the results we obtained. The data was collected using OpenSearch Benchmark to run the Big5 workload against different distributions and configurations of OpenSearch. In particular:
+
+- We used OpenSearch Benchmark---our standard benchmarking tool---for testing. Rally might produce different results.
+- We ran each test against a single-node cluster to ensure better reproducibility and easier setup.
+- The instance type was *c5.2xlarge*: 8 vCPU, 16 GB, an average instance type. Using oversized instances can conceal resource usage efficiency improvements.
+- The Big5 index was comprised of a single shard, no replicas (run with `--workload-params="number_of_replicas:0"`).
+- During the test, we ingested data using a single bulk indexing client to ensure that data is ingested in chronological order.
+- We ran the tests in [benchmarking mode](https://opensearch.org/docs/latest/benchmark/user-guide/target-throughput/#benchmarking-mode) (`target-throughput` was disabled) so that the OpenSearch client sends requests to the OpenSearch cluster as quickly as possible.
```

Comment:
```suggestion
- We ran the tests in [benchmarking mode](https://opensearch.org/docs/latest/benchmark/user-guide/target-throughput/#benchmarking-mode) (`target-throughput` was disabled) so that the OpenSearch client sent requests to the OpenSearch cluster as quickly as possible.
```